jazz_create-serverless-service/components/validate-arn.js

```diff
@@ -0,0 +1,26 @@
+/** 
+  @module: validate-arn.js
+  @description: Validates ARN value for a lambda function in this case Authorizer
+  @author:
+  @version: 1.0
+**/
+
+var validateARN = (arn) => {
+
+  var arnvalues = arn.split(":");
+  var isarnvalid = true;
+  //validate if arn is a valid Lambda
+  if(arn.indexOf("arn:aws:lambda") !== 0)
```

Comment:
- indentation "if ("
- why not use || and combine the if conditions

Reply:
Changed with OR